landbosse/model/CollectionCost.py

```diff
@@ -381,12 +373,19 @@ def calc_num_turb_partial_strings(num_leftover_turb, num_turb_per_cable):
                 turb_per_partial_string.append(0)
             num_remaining -= max_turb
 
-        # Calculate the percentage of full string turbines on a partial string
         perc_partial_string = np.divide(turb_per_partial_string, num_turb_per_cable)
 
-        return perc_partial_string
+        # Check to make sure there aren't any zeros in num_turbines_per_cable, which is used as the denominator
+        # in the division above. If there is a zero, then print a warning and change NaN to 0 in perc_partial_string.
+        if num_turb_per_cable.__contains__(0):
```

Comment:
Denominator is num_turb_per_cable and this should never be zero as far as I understand but we can check and throw a warning if it is.